readme.md

```diff
@@ -120,6 +120,7 @@ Click the type names for complete docs.
 - [`SetReturnType`](source/set-return-type.d.ts) - Create a function type with a return type of your choice and the same parameters as the given function type.
 - [`Asyncify`](source/asyncify.d.ts) - Create an async version of the given function type.
 - [`Simplify`](source/simplify.d.ts) - Flatten the type output to improve type hints shown in editors.
+- [`IsJsonable`](source/is-jsonable.d.ts) - Transforms type T into a type that can be structurally matched to [`JsonValue`](source/basic.d.ts) type alias
```

Comment:
Needs a simpler description.

Reply:
How is "Transforms a type to a type that is assignable to `JsonValue` type". (This is what I wrote in most recent commits).

I am open to discussion.  It is kind of like a mapped type, but it does so recursively. And it is a kind of identity for primitives and primitive properties.  So I chose 'transforms'.   And I think it is simpler to say the new type is "assignable".

Reply:
> Transforms a type to a type that is assignable to JsonValue type

Sounds good, but `transforms` => `transform`.